server/service/ElasticsearchClientService.js

```diff
@@ -69,7 +69,7 @@ exports.getElasticsearchClientOperationalState = async function(uuid) {
 exports.getElasticsearchClientServiceRecordsPolicy = async function(uuid) {
   var value = await elasticsearchService.getElasticsearchClientServiceRecordsPolicyAsync(uuid);
   var response = {};
-  response['application/json'] = {
```

Comment:
This change is already part of PR https://github.com/openBackhaul/OamLog/pull/389/files which is fix for issue https://github.com/openBackhaul/OamLog/issues/387 we can remove changes here